FirebaseInAppMessaging/Swift/Tests/Unit/FirebaseInAppMessaging_APIBuildTests.swift

```diff
@@ -0,0 +1,309 @@
+    let _: URL? = bannerDisplay.actionURL
+
+    let campaignInfo: FirebaseInAppMessaging.InAppMessagingCampaignInfo! = nil
+    let _: String = campaignInfo.messageID
+    let _: String = campaignInfo.campaignName
+    let _: Bool = campaignInfo.renderAsTestMessage
+
+    let nonnullImageData = FirebaseInAppMessaging.InAppMessagingImageData(
+      imageURL: nonnullText,
+      imageData: Data()
+    )
+    let _: Data? = nonnullImageData.imageRawData
+    let _: String = nonnullImageData.imageURL
+
+    let nullableActionButton: FirebaseInAppMessaging.InAppMessagingActionButton? = nil
+    let cardDisplay = FirebaseInAppMessaging.InAppMessagingCardDisplay(
+      campaignName: nonnullText,
+      titleText: nonnullText,
+      bodyText: nullableText,
+      textColor: nonnullColor,
+      portraitImageData: nonnullImageData,
+      landscapeImageData: nullableImageData,
+      backgroundColor: nonnullColor,
+      primaryActionButton: button,
+      secondaryActionButton: nullableActionButton,
+      primaryActionURL: nullableURL,
+      secondaryActionURL: nullableURL,
+      appData: nullableDict
+    )
+    _ = cardDisplay as FirebaseInAppMessaging.InAppMessagingDisplayMessage
+    let _: String = cardDisplay.title
+    let _: String? = cardDisplay.body
+    let _: UIColor = cardDisplay.textColor
+    let _: FirebaseInAppMessaging.InAppMessagingImageData = cardDisplay.portraitImageData
+    let _: FirebaseInAppMessaging.InAppMessagingImageData? = cardDisplay.landscapeImageData
+    let _: UIColor = cardDisplay.displayBackgroundColor
+    let _: FirebaseInAppMessaging.InAppMessagingActionButton = cardDisplay.primaryActionButton
+    let _: URL? = cardDisplay.primaryActionURL
+    let _: FirebaseInAppMessaging.InAppMessagingActionButton? = cardDisplay.secondaryActionButton
+    let _: URL? = cardDisplay.secondaryActionURL
+
+    let displayMessage = FirebaseInAppMessaging.InAppMessagingDisplayMessage(
+      messageID: nonnullText,
+      campaignName: nonnullText,
+      renderAsTestMessage: true,
+      messageType: messsageType,
+      triggerType: triggerType
+    )
+    let _: FirebaseInAppMessaging.InAppMessagingCampaignInfo = displayMessage.campaignInfo
+    let _: FirebaseInAppMessaging.FIRInAppMessagingDisplayMessageType = displayMessage.type
+    let _: FirebaseInAppMessaging.FIRInAppMessagingDisplayTriggerType = displayMessage.triggerType
+    let _: [AnyHashable: Any]? = displayMessage.appData
+
+    let imageOnlyDisplay = FirebaseInAppMessaging.InAppMessagingImageOnlyDisplay(
+      campaignName: nonnullText,
+      imageData: nonnullImageData,
+      actionURL: nullableURL,
+      appData: nullableDict
+    )
+    _ = imageOnlyDisplay as FirebaseInAppMessaging.InAppMessagingDisplayMessage
+    let _: FirebaseInAppMessaging.InAppMessagingImageData = imageOnlyDisplay.imageData
+    let _: URL? = imageOnlyDisplay.actionURL
+
+    let modalDisplay = FirebaseInAppMessaging.InAppMessagingModalDisplay(
```

Comment:
Is `FirebaseInAppMessaging.` required or just added for clarity in this file? Or in other words, are there now namespace collisions if both `FirebaseInAppMessaging` and `FirebaseInAppMessagingSwift` are imported? (Sorry in advance, not very familiar with the FIAM API.)

Reply:
Perfect, thanks for explaining!